src/pages/iv-items/forms/ItemsForm.js

```diff
@@ -469,7 +480,9 @@ export default function ItemsForm({ labels, maxAccess: access, setStore, store,
                       <Checkbox
                         name='ivtItem'
                         checked={formik.values.ivtItem}
-                        onChange={formik.handleChange}
+                        onChange={event => {
```

Comment:
u can used formik.handleChange and in edit mode if value equal null u can use || false